packages/katana/components/AprModal.tsx

```diff
@@ -152,6 +154,26 @@ export function AprModal({isOpen, onClose, vault, apr}: TAprModal): ReactElement
 						<span className={'text-[16px] font-bold text-white'}>{'Expected Net APR'}</span>
 						<span className={'text-[16px] font-bold text-white'}>{toPercent(totalAPR)}</span>
 					</div>
+					{steerRewardPoints !== undefined && steerRewardPoints > 0 && (
+						<div>
+							<p className={'text-regularText text-left text-sm leading-relaxed'}>
+								{'This vault earns '}
+								{formatAmount(steerRewardPoints, 2, 2)}
+								{' Steer Points / dollar deposited,'}
+							</p>
+							<p className={'text-regularText text-left text-sm leading-relaxed'}>
+								{'...But you must '}
```

Comment:
looks like a typo here. `{'...But you must '}` should be {'but you must '}

<img width="1362" height="924" alt="Image" src="https://github.com/user-attachments/assets/0068a060-8fe3-4af1-becb-521391d7e19f" />